app/__main__.py

```diff
@@ -71,11 +71,18 @@ def __init__(self, *, intents: discord.Intents) -> None:
     async def setup_hook(self) -> None:
         """Run async setup code before our bot connects.
 
-        Right now this just synchronizes our application commands with Discord.
+        Synchronizes our application commands with Discord and sets up the bot's description.
         """
         self.add_view(UpgradeView())
         await self.tree.sync()
 
+        await (await self.application_info()).edit(
+            description=(
+                "Enable a channel with </config enable:1266082362345656333>"
+                "and use </send:1266082362345656332> to send messages!"
```

Comment:
The IDs will change per bot instance. We should be able to get them from the command tree though.